omod/src/test/resources/test-hibernate.cfg.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE hibernate-configuration PUBLIC
+	"-//Hibernate/Hibernate Configuration DTD 3.0//EN"
+	"http://www.hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+	<session-factory>
```

Comment:
How did you figure this out? 😊

Reply:
Thoroughly searched the openmrs site:

https://wiki.openmrs.org/display/docs/Unit+Testing+with+Database+Data
https://wiki.openmrs.org/display/docs/Unit+Tests
https://wiki.openmrs.org/display/docs/Module+Unit+Testing

Reply:
That was clever of you! 👍